tests/python/test_solana.py

```diff
@@ -1,65 +1,36 @@
 from ragger.backend import RaisePolicy
-from ragger.navigator import NavInsID, NavIns
 from ragger.utils import RAPDU
 
 from .apps.solana import SolanaClient, ErrorType
 from .apps.solana_cmd_builder import SystemInstructionTransfer, Message, verify_signature, OffchainMessage
 from .apps.solana_utils import FOREIGN_PUBLIC_KEY, FOREIGN_PUBLIC_KEY_2, AMOUNT, AMOUNT_2, SOL_PACKED_DERIVATION_PATH, SOL_PACKED_DERIVATION_PATH_2, ROOT_SCREENSHOT_PATH
-from .apps.solana_utils import enable_blind_signing, enable_short_public_key, enable_expert_mode, navigation_helper_confirm, navigation_helper_reject
+from .apps.solana_utils import enable_blind_signing, enable_short_public_key, enable_expert_mode
 
 
 class TestGetPublicKey:
 
-    def test_solana_get_public_key_ok(self, backend, navigator, test_name):
+    def test_solana_get_public_key_ok(self, backend, scenario_navigator, navigator, test_name):
         sol = SolanaClient(backend)
         from_public_key = sol.get_public_key(SOL_PACKED_DERIVATION_PATH)
 
         with sol.send_public_key_with_confirm(SOL_PACKED_DERIVATION_PATH):
-            if backend.firmware.device.startswith("nano"):
-                navigator.navigate_until_text_and_compare(NavInsID.RIGHT_CLICK,
-                                                          [NavInsID.BOTH_CLICK],
-                                                          "Approve",
-                                                          ROOT_SCREENSHOT_PATH,
-                                                          test_name)
-            else:
-                instructions = [
-                    NavInsID.USE_CASE_REVIEW_TAP,
-                    NavInsID.USE_CASE_ADDRESS_CONFIRMATION_CONFIRM,
-                    NavInsID.USE_CASE_STATUS_DISMISS
-                ]
-                navigator.navigate_and_compare(ROOT_SCREENSHOT_PATH,
-                                               test_name,
-                                               instructions)
+            scenario_navigator.address_review_approve(path=ROOT_SCREENSHOT_PATH)
```

Comment:
Didn't you forget `test_name`? Or is it always the default name?
Else, you may not need it the the test function definition.

Reply:
Nope. The `scenario_navigator` fixture is instanciated with the default `test_name`, no need to input it again if we don't change it.